packages/now-build-utils/src/detect-builders.ts

```diff
@@ -25,6 +25,7 @@ const MISSING_BUILD_SCRIPT_ERROR: ErrorResponse = {
 // Static builders are special cased in `@now/static-build`
 function getBuilders({ tag }: Options = {}): Map<string, Builder> {
   const withTag = tag ? `@${tag}` : '';
+  const config = { zeroConfig: true };
```

Comment:
What's the difference between this config and the one on line 16?

Reply:
That it's not a reference

Reply:
Is the one on line 16 still used?

Reply:
Yup, it is, for the static file builders, otherwise TypeScript would probably complain